src/Commands/DibsCommand.php

```diff
@@ -273,13 +276,18 @@ protected function getDibsReport($site, $regex) {
         $envStatus = $dibs['for'] === $env ? 'Already called' : 'Available';
       }
 
-      $status[] = [
-        'env' => $env,
-        'status' => $envStatus,
-        'by' => isset($dibs['by']) ? $dibs['by'] : NULL,
-        'at' => isset($dibs['at']) ? date('D M jS \a\t h:ia', $dibs['at']) : NULL,
-        'message' => isset($dibs['message']) ? $dibs['message'] : NULL,
-      ];
+      // Further filter report by the age the of the dibs if a threshold was set.
+      $age = isset($dibs['at']) ? time() - isset($dibs['at']) : 0;
```

Comment:
Was trying to get a case where I could filter out some, but not all older dibs envs, but never could, even with absurdly long `older-than` thresholds.  I think it's the `time() - isset($dibs['at'])`, which PHP would translate into `time() - 1` pretty much every time.

Reply:
Oh, of course! testing++